src/CMakeLists.txt

```diff
@@ -223,8 +223,13 @@ else(INTEL_SSE4_1)
   message(STATUS "Skipping target ec_jerasure_sse4 & ec_shec_sse4: -msse4.1 not supported")
 endif(INTEL_SSE4_1)
 
+set(EXTRALIBS rt ${CMAKE_DL_LIBS} ${ATOMIC_OPS_LIBRARIES})
+if(LINUX)
+  set(LIB_RESOLV resolv)
+  set(LIB_UUID uuid)
```

Comment:
i don't think we are using libuuid anymore. see https://github.com/ceph/ceph/pull/10391/commits/3170f16e864e257416bccdd73777bff775767a8f i am planning to drop it in cmake.